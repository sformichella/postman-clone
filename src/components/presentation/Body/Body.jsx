import React from "react";

export default function Body() {
  return (
    <>
      <label htmlFor="body"></label>
      <input
        id="body"
        type="text"
        defaultValue="Enter request body here."
      />
    </>
  );
}
