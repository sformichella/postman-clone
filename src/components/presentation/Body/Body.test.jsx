import React from "react";
import { render, cleanup } from "@testing-library/react";
import Body from "./Body";

describe("Body component", () => {
  afterEach(() => cleanup());
  it("renders Body", () => {
    const { asFragment } = render(<Body />);
    expect(asFragment()).toMatchSnapshot();
  });
});
