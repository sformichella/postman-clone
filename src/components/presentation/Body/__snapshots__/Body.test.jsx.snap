// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Body component renders Body 1`] = `
<DocumentFragment>
  <label
    for="body"
  >
    Body: 
  </label>
  <input
    id="body"
    name="body"
    placeholder="Enter request body here."
    type="text"
  />
</DocumentFragment>
`;
